packages/@aws-cdk/aws-fsx/test/lustre-file-system.test.ts

```diff
@@ -35,15 +35,15 @@ describe('FSx for Lustre File System', () => {
       vpcSubnet,
     });
 
-    expectCDK(stack).to(haveResource('AWS::FSx::FileSystem'));
-    expectCDK(stack).to(haveResource('AWS::EC2::SecurityGroup'));
+    Template.fromStack(stack).hasResource('AWS::FSx::FileSystem', {});
+    Template.fromStack(stack).hasResource('AWS::EC2::SecurityGroup', {});
```

Comment:
Not a big deal, but how do you feel about doing `Template.fromStack(stack).resourceCountIs('AWS::FSx::FileSystem', 1);` 
for a lot of these.